_sass/custom/_footer.scss

```diff
@@ -1,52 +1,58 @@
-
 .footer {
-    padding-top: 50px;
-    padding-bottom: 50px;
-    background-color: $primary-30;
-    padding-top: 40px;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  background-color: $primary-30;
+  padding-top: 40px;
+  text-align: left;
+  color: $white;
+  min-height: 200px;
+
+  .follow-us-title {
```

Comment:
I think here, instead of new class you could just use the tag `h2`, but considering the h2 already exist in the page, I am not sure this change is correct. Please @lilyndk could you have a look?

Reply:
Yes, agreed that we should just use h2 here instead of introducing a new class. Thanks @MitaliShah

Reply:
Got it! Thank you for confirming, @lilyndk